DDECal/test/tRotationConstraint.cc

```diff
@@ -48,11 +49,13 @@ BOOST_AUTO_TEST_CASE( test_rotation )
   }
 }
 
-BOOST_AUTO_TEST_CASE( test_rotation_and_diagonal )
+BOOST_DATA_TEST_CASE( test_rotation_and_diagonal,
+                      boost::unit_test::data::make({ true, false }),
+                      doPhaseReference )
 {
   RotationAndDiagonalConstraint constraint;
   constraint.InitializeDimensions(1, 1, 1);
-  constraint.SetDoPhaseReference(true);
+  constraint.SetDoPhaseReference(doPhaseReference);
```

Comment:
So sorry, this should have been `doRotationReference`...

Reply:
Should I also update the parset key in DDECal.cc?

Reply:
Yes, but only for this particular constraint; for the existing constraints (tec etc) it should stay phasereference.

Reply:
So you should add a parset key `rotationreference`.